rule-examples/src/main/java/com/solibri/smc/api/examples/beginner/UIComponentExample.java

```diff
@@ -167,12 +186,75 @@ public UIContainer getParametersUIDefinition() {
 		 */
 		mainContainer.addComponent(createContainerWithLabelAndRadioButtonWithImages());
 
+		mainContainer.addComponent(createCustomStringParameterComponent());
```

Comment:
Could add an empty row after this before the next comment.

Reply:
I am not editing these files manually @Jussi-Saravuo It's generated from api documentation.